src/utils/usageReportConsent.js

```diff
@@ -7,3 +7,7 @@ export const getConsentStatus = () => {
 export const setConsentStatus = consent => {
   localStorage.setItem(KEY, !!consent)
 }
+
+export const getConsentConfirmationStatus = () => {
+  return (localStorage.getItem(KEY) === null || localStorage.getItem(KEY) === undefined)
```

Comment:
when there is no setting, `null` is given, do you think that's enough?

after the setting the key will be boolean, so no way to get it undefined.

Reply:
Right exactly, that's the idea. When `null` is returned because there is no `usage-data-consent` setting, then the consent popup is shown. I tested this behavior on my banana phone and it worked as expected. The undefined part is there just in case honestly, we could probably do away with it

Reply:
then what about this instead of checking null and undefined.

```
return !localStorage.getItem(KEY)
```

Reply:
I think it's best to be precise here. `getItem` on a key that was never set returns `null`. That's what we should check.

https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem

Reply:
Anyone in favor of removing `|| localStorage.getItem(KEY) === undefined` from the if-statement? That is, leaving it precisely as:

```
  return (localStorage.getItem(KEY) === null)
```